apps/src/music/MusicRegistry.ts

```diff
@@ -12,6 +12,7 @@ class MusicRegistry {
 
   public showSoundFilters: boolean = false;
   public showSoundsPanelInSoundsMode: boolean = false;
+  public sortCdoSoundsByType: boolean = false;
```

Comment:
Curious what to call these sounds... `CdoSounds` is a new term, but `NonRestricted` or `NonRestrictedPacks` might make more sense given the way the `MusicLibrary` currently refers to them.

Reply:
`UnrestrictedPacks` maybe?

Reply:
Oh, that should work well!

Reply:
That works - `default` could be an option too since that's how we refer to not choosing a pack?